guacamole/src/main/webapp/translations/es.json

```diff
@@ -260,16 +282,16 @@
         "ACTION_DELETE"      : "@:APP.ACTION_DELETE",
         "ACTION_SAVE"        : "@:APP.ACTION_SAVE",
 
-        "DIALOG_HEADER_CONFIRM_DELETE" : "Eliminar perfil de compartir",
+        "DIALOG_HEADER_CONFIRM_DELETE" : "Eliminar perfil compartido",
         "DIALOG_HEADER_ERROR"          : "@:APP.DIALOG_HEADER_ERROR",
 
         "FIELD_HEADER_NAME"               : "Nombre:",
         "FIELD_HEADER_PRIMARY_CONNECTION" : "Conexión primaria:",
 
-        "SECTION_HEADER_EDIT_SHARING_PROFILE" : "Editar perfil de compartir",
+        "SECTION_HEADER_EDIT_SHARING_PROFILE" : "Editar perfil compartido",
```

Comment:
The English name for this Guacamole concept, "sharing profile", refers to a set of configuration parameters that describe how the related connection can be shared. It is a profile (set of parameters) that describes sharing (the type of action performed by the user on the connection).

I'm not a Spanish speaker, but as far as I can tell, "compartido" is an adjective that means "shared". In English, "shared profile" would be incorrect, as it's not the profile itself that is shared.

For example:

* "_Sharing_ profile" - A profile describing how sharing may be performed (correct).
* "_Shared_ profile" - A profile that is shared with others (incorrect).

Does the new text have the correct meaning?

Reply:
Thanks for the explanation of sharing profile. I have reverted this to "perfil de compartir" as "perfil compartido" is not the right one for this meaning...  Already updated